### DISCUSSION TITLE
Can I whitelist outgoing email to specific email addresses?

### DISCUSSION THREAD

**Discussion by snapet** [rspamd/rspamd#5253] [Q&A] [answered]
Hi,

Very grateful for this software it saves me tons of time and having to watch rubbish.

I live in Belgium where we can report phishing mails to safeonweb.be. They track phishing mails and report these phishers and the mails to various companies that use this information to ban and/or train them I suppose. I quite rigorously like to report these.

I use rspamd as milter together with Postfix. Now, since using rspamd with dkim signing I have the scanning of outgoing mail enabled and notice now that my mails fail to go to safeonweb, so I can no longer report them. That is most unfortunate but also made me think: what if I send something to a regular contact that somehow is deemed spam of phishing or whatever?
I tried to find a solution online but mostly find entire domain whitelisting which is not what I want to do. I feel I still need to avoid my systems starting to spam if they would become compromised, so that doesn't feel like the standard solution anyway.

Thanks for any help or pointers!
(and sorry if I failed to find the obvious)

Best,
peter

**Comment by snapet** [comment]
Bump, or nudge perhaps. :-)

**Comment by fatalbanana** [comment, marked as answer]
Likely way to do it is using [settings module](https://rspamd.com/doc/configuration/settings.html):

~~~hcl
whitelist {
	priority = low;
	rcpt = "postmaster@example.com";
	want_spam = yes;
}
~~~

It will apply to all messages with this recipient.

To target messages with *only* this recipient (and not those which have been batched with any other recipient) you might use `selector` instead of `rcpt`:

~~~hcl
	selector = 'rcpts(smtp):addr.first.equal(postmaster@example.com);rcpts(smtp):addr.last.equal(postmaster@example.com)';
~~~

**Comment by snapet** [comment]
Thanks for this. I'm actually a bit embarrassed it was that easy.
